src/components/discovery/VariantSearchHeader.js

```diff
@@ -14,15 +17,24 @@ const VariantSearchHeader = ({dataType, addVariantSearchValues}) => {
     const labelCol = {lg: { span: 24 }, xl: { span: 4 }, xxl: { span: 3 }};
     const wrapperCol = {lg: { span: 24 }, xl: { span: 20 }, xxl: { span: 18 }};
 
+    // obtain ref and alt values from state
+    const form = useSelector(state =>
+        state.explorer?.dataTypeFormsByDatasetID[Object.keys(state.explorer.dataTypeFormsByDatasetID)[0]]);
+    const activeRefValue = form[0].formValues?.conditions === undefined ? "" : form[0].formValues.conditions.filter(c =>
+        c.value.field === "[dataset item].reference"
+    )[0].value.searchValue;
+    const activeAltValue = form[0].formValues?.conditions === undefined ? "" : form[0].formValues.conditions.filter(c =>
+        c.value.field === "[dataset item].alternative"
+    )[0].value.searchValue;
     const handleAssemblyIdChange = (value) => {
 
         addVariantSearchValues({assemblyId: value});
 
-    // temp workaround for bug in Bento back end:
-    // files ingested with GRCh37 reference need to be searched using assemblyId "Other"
+      // temp workaround for bug in Bento back end:
+      // files ingested with GRCh37 reference need to be searched using assemblyId "Other"
 
-    // so if assembly is "Other", pass that value to the form, but use
-    // "GRCh37" as the reference for gene lookup in Gohan
+      // so if assembly is "Other", pass that value to the form, but use
+      // "GRCh37" as the reference for gene lookup in Gohan
         if (value === "Other") {
             setLookupAssemblyId("GRCh37");
             return;
```

Comment:
this workaround is no longer needed, these lines can be deleted

Reply:
fixed on another branch